titus_isolate/predict/resource_usage_predictor.py

```diff
@@ -91,7 +91,7 @@ class ResourceUsagePredictor(SimpleCpuPredictor):
     def __translate_usage(usages: Dict[str, List[float]]) -> dict:
         out_usage = {}
         for resource_name, values in usages.items():
-            out_usage[RESOURCE_HEADING_MAPPINGS[resource_name]] = values[:60]
+            out_usage[RESOURCE_HEADING_MAPPINGS[resource_name]] = values[-60:]
```

Comment:
why not doing it [here](https://github.com/Netflix-Skunkworks/titus-isolate/blob/b22a07bb9a51c27250ef8289561114bbb0a281d2/titus_isolate/monitor/utils.py#L162) instead, so it covers all the calls to [get_pcp_usage](https://github.com/Netflix-Skunkworks/titus-isolate/blob/b22a07bb9a51c27250ef8289561114bbb0a281d2/titus_isolate/monitor/workload_monitor_manager.py#L44)?

Reply:
The 60 elements limit is a constraint of clients that `titus-isolate` is calling, not of `titus-isolate` or of the resource usage collectors.

Reply:
it's also a constraint that what's logged matches how it's passed to clients. hence wanting to put it as upstream as possible so it's less error-prone.